crud/request.go

```diff
@@ -0,0 +1,109 @@
+package crud
+
+import (
+	"context"
+
+	"github.com/tarantool/go-tarantool"
+)
+
+type baseCrudRequest struct {
```

Comment:
I don't think it makes sense in such naming in the submodule:
```suggestion
type baseRequest struct {
```

Reply:
Done.